### PR TITLE
test unified coins config from coins repo

### PR DESCRIPTION
**Pull request by smk762**
For testing unified coin config generated from coins repo.
ref: https://github.com/KomodoPlatform/coins/pull/502

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -79,7 +79,11 @@ FetchContent_Declare(
 )
 
 FetchContent_Declare(jl777-coins
+<<<<<<< HEAD
```
Comment (tonymorony) [inline CMakeLists.txt:82]: ?

Comment (smk762) [reply CMakeLists.txt]: fixed. forgot to save file :facepalm:

**Review by SirSevenG — DISMISSED** [review]
lgtm

**Comment by tonymorony** [comment]
@smk762 

```
Conflicting files
assets/config/0.5.6-coins.json
```

**Comment by Canialon** [comment]
hm i cant activate zombie 
image
<details><summary>error message</summary>
```
Please check your internet connection (e.g. VPN service or firewall might block it).

{
    "id": null,
    "mmrpc": "2.0",
    "result": {
        "details": {
            "error": "Error on platform coin ZOMBIE creation: EmptyLightwalletdUris",
            "error_data": {
                "error": "EmptyLightwalletdUris",
                "ticker": "ZOMBIE"
            },
            "error_path": "lib.z_coin_activation.z_coin",
            "error_trace": "lib:93] z_coin_activation:192] z_coin:792]",
            "error_type": "CoinCreationError"
        },
        "status": "Ready"
    }
}
```
</details>

**Comment by Canialon** [comment]
Hm, most of the explorer links show 404 error: kmd, doge, movr (receive link doesnt work at all), ftm, firo, qtum, arrr..

**Review by cipig — DISMISSED** [review]

**Review by cipig — APPROVED** [review]

**Review by cipig — APPROVED** [review]

**Comment by cipig** [comment]
> Clicking on explorer links in top right of tx details, or within swap progress doesn't do anything here - no borwser window opens as expected.

explorer links working fine here
image
clicking on the links opens https://explorer.btcz.rocks/tx/6d479ce7cca50582bf27c7a9d5a06a777df48a40c207103a011f1189d4a1b190
which coin did you test? maybe just wrong URL...

**Review by SirSevenG — APPROVED** [review]
lgtm

**Review by SylEze — APPROVED** [review]

**Pull request merged**